### ISSUE TITLE
ckf-1.7 branch doesn't check if public_url is configured

### ISSUE THREAD

**Issue by orfeas-k** [canonical/oidc-gatekeeper-operator#107]
As explained also [here](https://github.com/canonical/bundle-kubeflow/issues/669), `1.7/edge` channel uses [revision 250](https://github.com/canonical/oidc-gatekeeper-operator/releases/tag/rev250). Taking a look at rev250 code, we see that the [charm code](https://github.com/canonical/oidc-gatekeeper-operator/blob/rev250/src/charm.py) has the [_check_public_url](https://github.com/canonical/oidc-gatekeeper-operator/blob/rev250/src/charm.py#L133) method, but it seems like it is not called anywhere. Thus, the charm will never catch that the configuration is not there.

 ### Solution
Implement following in `ckf-1.7/track`
```diff
diff --git a/src/charm.py b/src/charm.py
index 4378d31..375037f 100755
--- a/src/charm.py
+++ b/src/charm.py
@@ -42,6 +42,7 @@ class Operator(CharmBase):
 
             image_details = self._check_image_details()
 
+            self._check_public_url()
         except CheckFailed as error:
             self.model.unit.status = error.status
             return
```

**Comment by orfeas-k** [comment]
Related to https://github.com/canonical/oidc-gatekeeper-operator/issues/18

**Comment by orfeas-k** [comment]
Closed by #108
